backend/lib/rpc-service/service.js

```diff
@@ -0,0 +1,91 @@
+import { RPC_URL } from '../config.js'
+import { base64pad } from 'multiformats/bases/base64'
+import { encode as cborEncode } from '@ipld/dag-cbor'
+import { decode as jsonDecode } from '@ipld/dag-json'
+import { request } from 'undici'
+import { rawEventEntriesToEvent } from './utils.js'
+import { Value } from '@sinclair/typebox/value'
+import { ClaimEvent, RawActorEvent, BlockEvent } from './data-types.js'
+
+/** @import {CID} from 'multiformats' */
+
+/**
+ * @param {string} method
+ * @param {Object} params
+  * @returns {Promise<object>}
+  */
```

Comment:
Let's fix the formatting, please.

```suggestion
 * @param {Object} params
 * @returns {Promise<object>}
 */
```

Reply:
Should be caught by linter or ignored?

Reply:
Although, here this style is obviously not intentional, so should be fixed independently of the linter result